src/test/selftest-parallel

```diff
@@ -13,7 +13,7 @@ if [[ -z "$RUN_PARTITIONS" ]]; then
     RUN_PARTITIONS=$(seq 0 $((NUM_PARTITIONS-1)));
 fi
 if [[ -z "$TEST_SPEC" ]]; then
-    TEST_SPEC="~[.]" # All non-hidden tests by default
```

Comment:
also need to update `CONTRIBUTING.md` (running tests) so that people know how to run acceptance test

Reply:
It is easy to not miss adding [.] to each [acceptance] test. Disabling [acceptance] tests in this script is better, because it is done only once and in one place.